src/combat.mjs

```diff
@@ -1,23 +1,154 @@
 import {item_roll, Logger} from "./util.mjs";
 import {socket} from "./module.mjs";
-import {ID_MAP, MODULE_ID} from "./constants.mjs";
+import {COMBAT_HEALTH_ESTIMATE, ID_MAP, MODULE_ID} from "./constants.mjs";
+import { actor_to_discord_ids } from "./sync.mjs";
 
 
-//TODO: Include some logic to set/remove these hooks when combat_tracking_enabled is changed
```

Comment:
This still needs to be done. There should be code that constructs or removes hooks if they exist with respect to `guild.combat_tracking_enabled` . This code should be run on startup and additionally when `guild.combat_tracking_enabled` is toggled.
Additionally, `open_socket_with_oronder()` may be called multiple times which means `set_combat_hooks() ` (also, `monks_token_bar_hooks()`) may potentially register duplicate hooks. I'll address these issues when I can, but you're also welcome to take a crack at them.

Reply:
Updated for the combat hooks. Take a look and see if it's how you want to handle it. Feel free to change it too :)